pkg/base/linuxutils/fs/fs_helper.go

```diff
@@ -102,10 +105,14 @@ func (h *WrapFSImpl) GetFSSpace(src string) (int64, error) {
 				/dev       7982M
 	*/
 
+	evalDuration := common.SystemCMDDuration.EvaluateDuration(prometheus.Labels{
+		"name":   strings.TrimSpace(fmt.Sprintf(CheckSpaceCmdImpl, "")),
+		"method": "GetFSSpace"})
 	stdout, _, err := h.e.RunCmd(fmt.Sprintf(CheckSpaceCmdImpl, src))
 	if err != nil {
 		return 0, err
 	}
+	evalDuration()
```

Comment:
if an error occurs - we are ignoring the duration?

Reply:
I don't see the point of measuring duration of the failed command

Reply:
Why? Some operations can take time and eventually fail.
Also, besides duration, we also count the number of operations.

Reply:
I agree with Andrey, I think we should collect metrics for failed calls also.

Reply:
Fixed